framework-test/src/main/java/org/checkerframework/framework/test/TestConfigurationBuilder.java

```diff
@@ -73,6 +78,13 @@ public static TestConfigurationBuilder getDefaultConfigurationBuilder(
                 .addOption("-implicit:class")
                 .addOption("-classpath", classPath);
 
+        // -Anomsgtext is needed to ensure expected errors can be matched, which is the
```

Comment:
How about moving this up to the earlier `addOption` calls, after `.addOption("-AnoJreVersionCheck")`?
Or maybe as separate statements after that, to make the long comments easier to read?
It would be nice if we had all the `-A` flags close together.

Reply:
Sounds good! I've moved the statements.